samples/notebooks/prediction/SDK_Custom_Predict.ipynb

```diff
@@ -0,0 +1,1269 @@
+        "%cd src_dir/\n",
+        "\n",
+        "import pickle\n",
+        "\n",
+        "import joblib\n",
+        "from preprocess import MySimpleScaler\n",
+        "from sklearn.datasets import load_iris\n",
+        "from sklearn.ensemble import RandomForestClassifier\n",
+        "\n",
+        "iris = load_iris()\n",
+        "scaler = MySimpleScaler()\n",
+        "\n",
+        "X = scaler.preprocess(iris.data)\n",
+        "y = iris.target\n",
+        "\n",
+        "model = RandomForestClassifier()\n",
+        "model.fit(X, y)\n",
+        "\n",
+        "joblib.dump(model, \"../model_artifacts/model.joblib\")\n",
+        "with open(\"../model_artifacts/preprocessor.pkl\", \"wb\") as f:\n",
+        "    pickle.dump(scaler, f)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3849066a33bd"
+      },
+      "source": [
+        "### Upload model artifacts and custom code to Cloud Storage\n",
+        "\n",
+        "Before you can deploy your model for serving, Vertex AI needs access to the following files in Cloud Storage:\n",
+        "\n",
+        "* `model.joblib` (model artifact)\n",
+        "* `preprocessor.pkl` (model artifact)\n",
+        "\n",
+        "Run the following commands to upload your files:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ca67ee52d4d9"
+      },
+      "outputs": [],
+      "source": [
+        "%cd ..\n",
+        "!gsutil cp model_artifacts/* {BUCKET_NAME}/{MODEL_ARTIFACT_DIR}/\n",
+        "!gsutil ls {BUCKET_NAME}/{MODEL_ARTIFACT_DIR}/"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "480a1d88ecdb"
+      },
+      "source": [
+        "## Build a custom serving container using the CPR model server\n",
+        "\n",
+        "Now that the model and processor has been trained and saved, it's time to build the custom serving container. Typically building a serving container requires writing model server code. However, with the Custom Prediction Routine feature, Vertex AI Prediction has published a [model server](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/model_server.py) that can be used out of the box.\n",
+        "\n",
+        "A custom serving container contains the follow 3 pieces of code:\n",
+        "1. [Model server](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/model_server.py)\n",
+        "  * HTTP server that hosts the model\n",
+        "  * Responsible for setting up routes/ports/etc.\n",
+        "  * In this example we will use the `google.cloud.aiplatform.prediction.model_server.ModelServer` out of the box.\n",
+        "1. [Request Handler](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/handler.py)\n",
```

Comment:
The numbers are not updated to 2 or 3 from what I see in the notebook.

Reply:
It renders correctly in Vertex AI Workbench and Colab. What are you using to render?

Reply:
Vertex Workbench https://screenshot.googleplex.com/p5JJPcQbG5KEP5y. The other parts look correct if there's no other sentences between the numbers.

Reply:
Ah, you're right... spacing issue, fixed.